tools/docker/Dockerfile

```diff
@@ -67,10 +67,11 @@ COPY start.sh /home/test/start.sh
 # Remove information on how to use sudo on login
 RUN sudo echo ""
 
-RUN git clone --depth=1 https://github.com/w3c/web-platform-tests.git
-
-RUN mkdir -p /home/test/.fonts && \
-    cp web-platform-tests/fonts/Ahem.ttf ~/.fonts && \
-    fc-cache -f -v
+RUN mkdir -p /home/test/web-platform-tests
```

Comment:
Can you add a comment explaining why this isn't just `git clone https://github.com/w3c/web-platform-tests` + `WORKDIR /home/test/web-platform-tests`?

And also comment that the reason for having a checkout as part of the image is to save on network traffic or speed things up?

Reply:
Having the checkout as part of the image was the previous state, but now we don't. It probably did make things a little faster, but I imagine the gains would decline as the image gets older and on the other hand it does make it much slower to iterate on the docker image. In the future we could maybe consider a task to build a base image containing a checkout, updated e.g. weekly.

Reply:
Oh, so the whole point of this now is to not run `git fetch`. I did not spot that, comment would be good :)

Reply:
Have the comment say "the remote is added but not fetched, the right revision is fetched inside the container".

But... then I have to wonder why bother setting this up in the Dockerfile at all, a plain git clone inside the container would be fewer lines of code and easier to understand.